atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/persistentlock/PersistentLockIsTakenException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.persistentlock;
+
+import java.util.Set;
+
+public class PersistentLockIsTakenException extends Exception {
+    private static final String MESSAGE = "Could not acquire the requested lock %s, "
```

Comment:
Change the message to take into account the `already open` case and a `different lock being released` case.